backend/src/schema/resolvers/embeds.js

```diff
@@ -11,16 +11,17 @@ export default {
     ...undefinedToNullResolver([
       'type',
       'title',
+      'description',
       'author',
       'publisher',
-      'date',
-      'description',
       'url',
+      'date',
       'image',
       'audio',
       'video',
       'lang',
-      'html',
```

Comment:
Hm, I think for the trusted oembed providers in `providers.json` we can return plain html to the client.